### PR TITLE
Initialize default Scope in TestDFNoTrace

### PR DESCRIPTION
**Pull request by wenleix**
Otherwise the test run with
```
python -m unittest -v
```

would fail.

### CODE REVIEW

File: torcharrow/test/test_trace.py [hunk 1]
```diff
@@ -45,6 +45,9 @@ def g(self, other):
 
 
 class TestDFNoTrace(unittest.TestCase):
+    def setUp(self):
+        Scope.default = Scope()
```
Comment (ejguan) [inline torcharrow/test/test_trace.py:49]: nit: If the scope is used by the whole class. You can use `setUpClass`. https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUpClass

It can also be achieved in a separate PR.

**Review by ejguan — APPROVED** [review]
LGTM

File: torcharrow/test/test_trace.py (hunk 1 above)
Comment (wenleix) [reply torcharrow/test/test_trace.py:49]: @ejguan : I see. The plan is  to deprecate `Scope` though: https://github.com/facebookresearch/torcharrow/pull/37

Comment (ejguan) [reply torcharrow/test/test_trace.py:49]: Then, no need to worry about it.
